src/internal.c

```diff
@@ -9870,9 +9876,9 @@ static int ProcessCSR(WOLFSSL* ssl, byte* input, word32* inOutIdx,
 
     *inOutIdx += status_length;
 
-    FreeOcspResponse(response);
```

Comment:
Frees the chain of single responses, if they were marked isDynamic. isDynamic isn't set here, so this is a no-op. The dynamically allocated single and status structs get freed directly below.